pages/running_tests.md

```diff
@@ -43,7 +43,7 @@ Integration tests are done with the Spring Test Context framework, and are locat
 This Spring test context will use a specific test database to run its tests:
 
 *   If you use an SQL database, JHipster will launch an in-memory H2 instance in order to use a temporary database for its integration tests. Liquibase will be run automatically, and will generate the database schema.
-*   If you use Cassandra, JHipster will launch a containerized version of Cassandra with Docker using [Testcontainers](https://www.testcontainers.org){:target="_blank" rel="noopener"}.
+*   If you use Cassandra or Redis, JHipster will launch a containerized version of Cassandra with Docker using [Testcontainers](https://www.testcontainers.org){:target="_blank" rel="noopener"}.
```

Comment:
Maybe add a dedicated list point (like we have for couchbase and neo for example).

Reply:
Ah, I got it wrong again. I've reverted it. I think we shouldn't add it here at all; the heading says, `This Spring test context will use a specific test database to run its tests`.